test/PublicApiTests/BaseLine/Microsoft.OData.PublicApi.net45.bsl

```diff
@@ -8237,7 +8236,6 @@ public class Microsoft.OData.Client.DataServiceContext {
     public virtual DataServiceQuery`1 CreateQuery (string entitySetName)
     public virtual DataServiceQuery`1 CreateQuery (string resourcePath, bool isComposable)
     public virtual DataServiceQuery`1 CreateSingletonQuery (string singletonName)
-    public virtual Microsoft.OData.Client.DataServiceResponse DeepInsert (T resource)

```

Comment:
Undo this change since it is unrelated to this PR

Reply:
One of previous commit forgot to update the bsl.